server/sonar-server/src/main/java/org/sonar/server/issue/ws/SearchAction.java

```diff
@@ -803,7 +798,7 @@ private void addZeroFacetsForSelectedItems(Request request, String facetName, Se
     List<String> requestParams = request.paramAsStrings(facetName);
     if (requestParams != null) {
       for (String param : requestParams) {
-        if (!itemsFromFacets.contains(param)) {
+        if (!itemsFromFacets.contains(param) && !IssueQueryService.LOGIN_MYSELF.equals(param)) {
```

Comment:
Here, the ``assignees`` parameter contains the ``__me__`` marker, which we don't want to appear in the assignees facet.